examples/ejb-remote-call/src/main/java/org/superbiz/ws/App.java

```diff
@@ -0,0 +1,23 @@
+package org.superbiz.ws;
```

Comment:
`package org.superbiz.ws` suggests this is a webservice, rather than a Remote EJB call.